src/queries/__tests__/labelText.test.tsx

```diff
@@ -46,6 +50,12 @@ test('getByLabelText, queryByLabelText, findByLabelText', async () => {
     <Section />
   );
 
+  expect(getByLabelText(INPUT_LABEL).props.accessibilityLabelledBy).toEqual(
```

Comment:
Would it be possible to get that in a different `test()` case? I know there was only one so far, but I prefer explicitly named `test` cases for the different edge cases we want to cover. Makes it easier to figure out the issue when a test starts failing.